common/views/components/InstallationPromo/InstallationPromo.js

```diff
@@ -1,29 +1,85 @@
 // @flow
 import {spacing, font} from '../../../utils/classnames';
+import {striptags} from '../../../utils/striptags';
+// import {formatDate, formatDateRangeWithMessage} from '../../../utils/format-date';
+import Image from '../Image/Image';
+// import Icon from '../Icon/Icon';
+// import {Fragment} from 'react';
+import type {Picture} from '../../../model/picture';
 
 type Props = {|
   id: string,
-  title: string
+  title: string,
+  description: string,
+  image: Picture,
+  start: Date,
+  end: ?Date
 |}
 
-const InstallationPromo = ({ id, title }: Props) => (
+function timeOrDescription(start, end, description) { // TODO not sure if it makes sense to show the dates?
+  // if (start && end) {
+  //   return (
+  //     <Fragment>
+  //       <time dateTime={start.toString()}>{formatDate(start)}</time>&mdash;<time dateTime={end.toString()}>{formatDate(end)}</time>
+  //     </Fragment>
+  //   );
+  // } else {
+  return striptags(description);
+  // }
+}
+
+// function statusColor(start, end, formatDateRangeWithMessage) {
+//   const dateMessageAndColor = formatDateRangeWithMessage(start, end);
+//   return dateMessageAndColor.color;
+// }
+
+// function statusText(start, end, formatDateRangeWithMessage) {
+//   const dateMessageAndColor = formatDateRangeWithMessage(start, end);
+//   return dateMessageAndColor.text;
+// }
+
+const InstallationPromo = ({ id, title, description, image, start, end }: Props) => (
   <a
     data-component='InstallationPromo'
     data-track-event={JSON.stringify({
       category: 'component',
       action: 'InstallationPromo:click'
     })}
     id={id}
-    href={`/installations/{id}`}
+    href={`/installations/${id}`}
     className='plain-link promo-link bg-cream rounded-corners overflow-hidden flex flex--column'>
-    <h2
-      className={`
-        promo-link__title
-        ${font({s: 'WB5'})}
-        ${spacing({s: 0}, {margin: ['top']})}
-      `}>
-      {title}
-    </h2>
+    <Image
```

Comment:
This is good for now - but let's maybe chat about how we deal with this further up the chain...